packages/beacon-node/src/api/impl/validator/index.ts

```diff
@@ -1063,7 +1063,11 @@ export function getValidatorApi({
       };
     },
 
-    async getAggregatedAttestation({attestationDataRoot, slot, committeeIndex}) {
+    async getAggregatedAttestation() {
+      throw new Error("Not implemented. Use getAggregatedAttestationV2 for now.");
```

Comment:
this will break if other validator clients call lodestar beacon node

Reply:
pls add TODO Electra here in the code